lib/Slider/Slider.js

```diff
@@ -579,6 +586,10 @@ module.exports = kind(
 			this.updatePopup(this.elasticFrom);
 			this.set('value',this.elasticFrom);
 
+			if (this.enableJumpIncrement) {
+				this.updateButtonStatus(v);
```

Comment:
Yes that's right. As additional color on the UX thinking, the user cannot tap on any of the jump increment buttons while dragging, so we are never in a scenario where it is a necessity to have these buttons disabled while dragging.